openshift-install-powervs

```diff
@@ -1422,6 +1477,12 @@ function main {
       warn "Enabling forceful destruction option for terraform destroy command"
       FORCE_DESTROY=1
       ;;
+    "-flavor")
+      shift
+      flavortype="$1"
+      vars+=" -var-file ../$varfile -var-file ../automation/compute-vars/$flavortype.tfvars"
+      process_flavor "$flavortype"
```

Comment:
This should be called before updating the vars.

Reply:
Updated this change